time_tracker_api/database.py

```diff
@@ -0,0 +1,28 @@
+"""
+Agnostic database assets
+
+Put here your utils and class independent of
+the database solution
+"""
+from flask import Flask
+
+RepositoryModel = None
+
+
+def init_app(app: Flask) -> None:
```

Comment:
Why not call this function 'init_database' directly ?

Reply:
There is some sort of standard that some objects like the APIs in flask Restplus have a function called init_app so they take the Flask app as a parameter and they do whatever initialization logic they need to be ready to be used by that app. So whatever module or package you add if you want to make it like pluggable to Flask this is pretty much the way to go